recipe/meta.yaml

```diff
@@ -26,6 +26,10 @@ requirements:
 
   host:
     - qt6-main {{ version }}
+    - libgl-devel                        # [linux]
+    - libglx-devel                       # [linux]
+    - libegl-devel                       # [linux]
+    - libopengl-devel                    # [linux]
```

Comment:
not sure if libopengl-devel is necessary tbh. we don't have it on qt-main

Reply:
```suggestion
```

Reply:
I'll try